### PR TITLE
Make Golioth Exploration its own module

### PR DESCRIPTION
**Pull request by szczys**
"Intro to Golioth" is the default module, directing users to API Training or Zephyr Training once completed.

### CODE REVIEW

File: website/docs/golioth-exploration/README.md
Comment (ChrisGammell) [inline website/docs/golioth-exploration/README.md:1]: Is this supposed to be a README.md file?

**Comment by ChrisGammell** [comment]
There are a couple messages in the training that say, "You will want to use hardware, but don't need it", but i don't think any alternative is offered?

**Comment by szczys** [comment]
Those messages are deprecated. The training is now focused on having hardware in hand. For the API training that's pre-compiled binaries for the nRF7002/9160 DKs. Same for the Zephyr training but attendees are building their own binaries.

**Review by ChrisGammell — APPROVED** [review]
LGTM

**Pull request merged**